sys/kern/signal.c

```diff
@@ -1,3 +1,5 @@
+#include "sys/siginfo.h"
```

Comment:
Please use convention for including header files that can be easily inferred by looking at source code of various kernel source code files.